public/app/core/services/echo/backends/analytics/GA4Backend.ts

```diff
@@ -11,28 +11,34 @@ declare global {
 
 export interface GA4EchoBackendOptions {
   googleAnalyticsId: string;
+  googleAnalytics4SendManualPageViews: boolean;
   user?: CurrentUserDTO;
 }
 
 export class GA4EchoBackend implements EchoBackend<PageviewEchoEvent, GA4EchoBackendOptions> {
   supportedEvents = [EchoEventType.Pageview];
-  trackedUserId: number | null = null;
+  googleAnalytics4SendManualPageViews = true;
 
   constructor(public options: GA4EchoBackendOptions) {
     const url = `https://www.googletagmanager.com/gtag/js?id=${options.googleAnalyticsId}`;
-    loadScript(url);
+    loadScript(url, true);
 
     window.dataLayer = window.dataLayer || [];
     window.gtag = function gtag() {
       window.dataLayer.push(arguments);
     };
     window.gtag('js', new Date());
 
-    const configOptions: Gtag.CustomParams = {};
+    const configOptions: Gtag.CustomParams = {
+      page_path: window.location.pathname,
+    };
 
     if (options.user) {
       configOptions.user_id = getUserIdentifier(options.user);
     }
+    if (!options.googleAnalytics4SendManualPageViews) {
+      this.googleAnalytics4SendManualPageViews = false;
+    }
```

Comment:
small nit, but this can probably just be simplified to 

```
      this.sendPageViews= options.googleAnalytics4SendManualPageViews;
```

or something? don't need to set it conditionally if we require it's always passed in

Reply:
Changed https://github.com/grafana/grafana/pull/57619/commits/4963aaaa72a57478eba5d1d6f01c232fafbec4cd